forge-gui/res/cardsfolder/c/caverns_of_despair.txt

```diff
@@ -5,4 +5,8 @@ K:No more than two creatures can attack each combat.
 S:Mode$ Continuous | GlobalRule$ No more than two creatures can block each combat. | Description$ No more than two creatures can block each combat.
 SVar:NonStackingEffect:True
 AI:RemoveDeck:Random
+DeckHints:Type$Planeswalker|Artifact|Enchantment
+SVar:NeedsToPlayVar:CountOpps GTCountMe
```

Comment:
Invalid SVar
but the logic also seems too simple, AI would no longer play this vs 100 creatures once it has 5+